aci/resource_aci_bfdmhnodepol.go

```diff
@@ -0,0 +1,252 @@
+package aci
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/ciscoecosystem/aci-go-client/v2/client"
+	"github.com/ciscoecosystem/aci-go-client/v2/models"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+func resourceAciBFDMultihopNodePolicy() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceAciBFDMultihopNodePolicyCreate,
+		UpdateContext: resourceAciBFDMultihopNodePolicyUpdate,
+		ReadContext:   resourceAciBFDMultihopNodePolicyRead,
+		DeleteContext: resourceAciBFDMultihopNodePolicyDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: resourceAciBFDMultihopNodePolicyImport,
+		},
+
+		SchemaVersion: 1,
+		Schema: AppendBaseAttrSchema(AppendNameAliasAttrSchema(map[string]*schema.Schema{
+			"tenant_dn": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"admin_state": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ValidateFunc: validation.StringInSlice([]string{
+					"disabled",
+					"enabled",
+				}, false),
+			},
+			"detection_multiplier": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+			"min_rx_interval": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+			"min_tx_interval": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+		})),
+	}
+}
+
+func getRemoteBFDMultihopNodePolicy(client *client.Client, dn string) (*models.BFDMultihopNodePolicy, error) {
+	bfdMhNodePolCont, err := client.Get(dn)
+	if err != nil {
+		return nil, err
+	}
+	bfdMhNodePol := models.BFDMultihopNodePolicyFromContainer(bfdMhNodePolCont)
+	if bfdMhNodePol.DistinguishedName == "" {
+		return nil, fmt.Errorf("BFDMultihopNodePolicy %s not found", dn)
```

Comment:
fmt.Errorf("BFDMultihopNodePolicy %s not found", dn) -> 
fmt.Errorf("BFD Multihop Node Policy %s not found", dn)